en/docs/apis/restapis/session.yaml

```diff
@@ -1,502 +1,632 @@
 tags:
-- name: me
-  description: Session management operations for the authenticated user.
-- name: admin
-  description: |
-    Operations available for privileged users, to be invoked on behalf of another user.
+  - name: me
+    description: Session management operations for the authenticated user.
+  - name: admin
+    description: >
+      Operations available for privileged users, to be invoked on behalf of another user.
+
+      **_These endpoints are not released in WSO2 Identity Server - 5.9.0 version and will be released in the next product release._**
+  - name: sysadmin
+    description: Operations available for privileged users, to be invoked for system management.
+
+# Applicable authentication mechanisms
+security:
+  - OAuth2: []
+  - BasicAuth: []
+
 paths:
   /me/sessions:
     get:
       tags:
-      - me
+        - me
+      description: >
+        A user can have multiple active sessions. This API retrieves information of all the active sessions of the authenticated user. <br>
+        <b>Permission required:</b> <br>
+        * None <br>
+        <b>Scope required:</b> <br>
+        * internal_login
       summary: Retrieve active sessions of the authenticated user
-      description: |
-        A user can have multiple active sessions. This API retrieves information of all the active sessions of the authenticated user. <br> <b>Permission required:</b> <br> * None <br> <b>Scope required:</b> <br> * internal_login
       operationId: getSessionsOfLoggedInUser
+      produces:
+        - application/json
       parameters:
-      - name: limit
-        in: query
-        description: |
-          Maximum number of records to return.
-          _This parameter is not supported yet._
-        schema:
-          type: integer
-          format: int32
-      - name: offset
-        in: query
-        description: |
-          Number of records to skip for pagination.
-          _This parameter is not supported yet._
-        schema:
-          type: integer
-          format: int32
-      - name: filter
-        in: query
-        description: |
-          Condition to filter the retrival of records.
-          _This parameter is not supported yet._
-        schema:
-          type: string
-      - name: sort
-        in: query
-        description: |
-          Define the order in which the retrieved records should be sorted.
-          _This parameter is not supported yet._
-        schema:
-          type: string
+        - $ref: '#/parameters/limitQueryParamNotImplemented'
+        - $ref: '#/parameters/offsetQueryParam'
+        - $ref: '#/parameters/filterQueryParamNotImplemented'
+        - $ref: '#/parameters/sortQueryParam'
       responses:
         200:
-          description: Successfully retrieved session information
-          content:
+          description: >
+            Successfully retrieved session information.
+            idpName will only be returned for federated authentication sessions.
```

Comment:
`idpName will only be returned for federated authentication sessions.`
Let's remove this part. This will only be available as a U2 update when doing the proactive patches.
Check other places as well

Reply:
Updated.